.snippets/text/node-operators/appchain-node/run-flags.md

```diff
@@ -1,4 +1,3 @@
-- `--name INSERT_NAME` - a human-readable name for this node
```

Comment:
isn't this still necessary, or no?

Reply:
just want to check on this before merging

Reply:
I would say yes because we want collators to provide a name for telemetry